kore/test/Test/Kore/Reachability/OnePathStrategy.hs

```diff
@@ -810,7 +810,7 @@ test_onePathStrategy =
             left' =
                 Pattern.withCondition
                     Mock.a
-                    (Condition.assign (inject Mock.x) Mock.a)
+                    (Condition.assign (inject Mock.x) Mock.a){substitution = mempty}
```

Comment:
This should just be `left' = Mock.a`. `Condition.assign` will just add that to the substitution, and `substitution = mempty` will delete it.

Reply:
Some definition-chasing later, I think the `{substitution = mempty}` just undoes whatever `Pattern.withCondition` does. I replaced it with `Pattern.fromTermLike` instead.